tests/library/browsercontext-route.spec.ts

```diff
@@ -250,19 +250,19 @@ it('should overwrite post body with empty string', async ({ context, server, pag
   expect(body).toBe('');
 });
 
-it('should chain continue', async ({ context, page, server }) => {
+it('should chain fallback', async ({ context, page, server }) => {
   const intercepted = [];
   await context.route('**/empty.html', route => {
     intercepted.push(1);
-    route.continue();
+    route.fallback();
   });
   await context.route('**/empty.html', route => {
```

Comment:
Should we add a test to make sure `fallback` works with the new [async handlers](https://github.com/microsoft/playwright/pull/14317)?

Reply:
Should be covered!